e2e/scripts/smoke_test.sh

```diff
@@ -108,15 +132,15 @@ rook_infra::cleanup() {
 { #try
 
     rook_infra::create
-    sleep 5
+    sleep 10    #need to add code to retry docker info till success
```

Comment:
can you add this check? seems like we need to fix these sleeps instead of letting them perpetuate